app/controllers/admin/users_controller.rb

```diff
@@ -100,7 +100,16 @@ def remove_role
   private
 
   def user_params
-    params.require(:user).permit(:name, :organization_id, :email, :password, :password_confirmation)
+    user_params = params.require(:user).permit(:name, :organization_id, :email, :password, :password_confirmation)
+    role_id = updated_role_id
+
+    user_params[:organization_role_join_attributes] = { role_id: role_id } if role_id
+
+    user_params
+  end
+
+  def updated_role_id
+    Role.find_by(resource_type: Role::TITLES[Role::ORG_USER], resource_id: params[:user][:organization_id], name: Role::ORG_USER)&.id
```

Comment:
Can this line be split up? It'd be a bit easier to read.

Reply:
Hi @dorner, thank you for the suggestions, can you please review the latest changes, thanks!